### PR TITLE
🌐 [i18n-KO] Translated `tasks/image_captioning.mdx` to Korean

### PR DESCRIPTION
**Pull request by sim-so**
# What does this PR do?

Translated the `tasks/image_captioning.mdx` file of the documentation to Korean.
Thank you in advance for your review!

- [x] 이미지 캡셔닝 Image captioning
  - [x] 포켓몬 BLIP 데이터셋 가져오기 Load the Pokémon BLIP captions dataset
  - [x] 데이터셋 전처리 Preprocess the dataset
  - [x] 기본 모델 가져오기 Load a base model
  - [x] 평가 Evaluate
  - [x] 학습! Train!
  - [x] 추론 Inference

Part of https://github.com/huggingface/transformers/issues/20179

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

This is a work on progress.
Could you review this PR when I finish this work?
@0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

### CODE REVIEW

**Comment by sim-so** [comment]
I quickly done it! 😅 
Would you review this PR?

**Review by 0525hhgus — APPROVED** [review]
좋은 번역 감사합니다 😄 문법까지 모두 완벽하네요..! 
~~전 다른 피드백 없습니다 👍~~ 하단에 리뷰 남겼습니다!

File: docs/source/ko/_toctree.yml [hunk 1]
```diff
@@ -91,8 +91,8 @@
       title: (번역중) Depth estimation
     title: (번역중) 컴퓨터 비전
   - sections:
-    - local: in_translation
-      title: (번역중) Image captioning
+    - local: tasks/image_captioning
+      title: (번역중) 이미지 캡셔닝
```
Comment (0525hhgus) [inline docs/source/ko/_toctree.yml:95]: `(번역중)` 태그가 아직 붙어 있습니다! 
의도하신 제목인지 확실치 않아서, 짧게 납깁니다..!

File: docs/source/ko/tasks/image_captioning.mdx [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+토큰을 입력하여 로그인하세요.
+
+
+```python
+from huggingface_hub import notebook_login
+
+notebook_login()
+```
+
+## 포켓몬 BLIP 캡션 데이터세트 가져오기[[load-the-pokémon-blip-captions-dataset]]
```
Comment (0525hhgus) [inline docs/source/ko/tasks/image_captioning.mdx:44]:
```suggestion
## 포켓몬 BLIP 캡션 데이터세트 가져오기[[load-the-pokmon-blip-captions-dataset]]
```
`load-the-pokmon-blip-captions-dataset`로 하시면, anchor가 작동할 것 같습니다!

**Review by 0525hhgus — CHANGES_REQUESTED** [review]
추가로 리뷰드립니다!

File: docs/source/ko/_toctree.yml (hunk 1 above)
Comment (sim-so) [reply docs/source/ko/_toctree.yml:95]: 의도한 것이 아니었어요! 😂 
확인 감사합니다!

File: docs/source/ko/tasks/image_captioning.mdx (hunk 2 above)
Comment (sim-so) [reply docs/source/ko/tasks/image_captioning.mdx]: 수정 제안 감사합니다!

**Review by wonhyeongseo — APPROVED** [review]
LGTM ❤️

**Comment by sim-so** [comment]
Happy Wednesday!
Could you review this PR? 😃
@sgugger, @ArthurZucker, @eunseojo

**Review by sgugger — APPROVED** [review]
Thanks a lot!

**Pull request merged**